NEWS.md

```diff
@@ -0,0 +1,3141 @@
+
+  Kristian Evers
+  Even Rouault
+  Kurt Schwehr
+  Mike Toews
+  Martin Desruisseaux
+  Charles Karney
+  Thomas Knudsen
+  Javier Goizueta
+  Bas Couwenberg
+  Adam Wulkiewicz
+  Aaron Puchert
+
+5.0.1 Release Notes
+-------------------
+
+BUG FIXES
+------------
+
+All bug fix numbers refer to issues or pull requests indexed at
+https://github.com/OSGeo/proj.4/
+
+* Handle ellipsoid change correctly in pipelines when
+  +towgs84=0,0,0 is set #881
+
+* Handle the case where nad_ctable2_init returns NULL #883
+
+* Avoid shadowed declaration errors with old gcc #880
+
+* Expand +datum properly +datum in pipelines #872
+
+* Fail gracefully when incorrect headers are encountered in grid
+  files #875
+
+* Improve roundtrip stability in pipelines using +towgs84 #871
+
+* Fixed typo in gie error codes #861
+
+* Numerical stability fixes to the geodesic package #826 #843
+
+* Make sure that transient errors are returned correctly #857
+
+* Make sure that locally installed header files are not used when
+  building PROJ #849
+
+* Fix inconsistent parameter names in proj.h/proj_4D_api.c #842
+
+* Make sure +vunits is applied #833
+
+* Fix incorrect Web Mercator transformations #834
+
+THANKS TO
+------------
+
+Version 5.0.1 is made possible by the following contributors:
+
+  Mike Toews
+  Kurt Schwehr
+  Even Rouault
+  Charles Karney
+  Thomas Knudsen
+  Kristian Evers
+
+
+5.0.0 Release Notes
+-------------------
+
+This version of PROJ introduces some significant extensions and
+improvements to (primarily) the geodetic functionality of the system.
+
+The main driver for introducing the new features is the emergence of
+dynamic reference frames, the increasing use of high accuracy GNSS,
+and the related growing demand for accurate coordinate
+transformations.  While older versions of PROJ included some geodetic
+functionality, the new framework lays the foundation for turning PROJ
+into a generic geospatial coordinate transformation engine.
+
+The core of the library is still the well established projection code.
+The new functionality is primarily exposed in a new programming
+interface and a new command line utility, "cct" (for "Coordinate
+Conversion and Transformation").  The old programming interface is
+still available and can - to some extent - use the new geodetic
+transformation features.
+
+The internal architecture has also seen many changes and much
+improvement.  So far, these improvements respect the existing
+programming interface. But the process has revealed a need to simplify
+and reduce the code base, in order to support sustained active
+development.
+
+!!!
+!!! Therefore we have scheduled regular releases over the coming years
+!!! which will gradually remove the old programming interface.
+!!!
+!!! This will cause breaking changes with the next two major version
+!!! releases, which will affect all projects that depend on PROJ
+!!! (cf. section "deprecations" below).
+!!!
```

Comment:
Perhas this can be expressed in a more markdown like manner?

Reply:
There's a similar thing further down

Reply:
I used a block. There is not so much options in markdown.